docs/release_notes/v1.7.0.md

```diff
@@ -0,0 +1,386 @@
+
+## New in this release
+
+### Dapr Runtime
+- **FIXED** Dapr setting a default content type of `text/plain` [4472](https://github.com/dapr/dapr/pull/4472)
+- **ADDED** Circuit Breaking [777](https://github.com/dapr/dapr/issues/777)
+- **ADDED** Content-Type for state operations [2892](https://github.com/dapr/dapr/issues/2892)
+- **ADDED** Resiliency policies across all building blocks [3586](https://github.com/dapr/dapr/issues/3586)
+- **CHANGED** Actor Reentrancy to stable feature [3811](https://github.com/dapr/dapr/issues/3811)
+- **CHANGED** gRPC Proxying to stable feature [3814](https://github.com/dapr/dapr/issues/3814)
+- **FIXED** Service invocation to return original status code on service invocation. Add the possibility to configure retries base on error type [3826](https://github.com/dapr/dapr/issues/3826)
+- **FIXED** Serialization of 64bit values. [3837](https://github.com/dapr/dapr/issues/3837)
+- **ADDED** Unsubscribe method for the configuration API. [3906](https://github.com/dapr/dapr/issues/3906)
+- **FIXED** Bug in runtime matching wildcard routes for pubsub. [3920](https://github.com/dapr/dapr/issues/3920)
+- **ADDED** Actor options configuration per actor type [3934](https://github.com/dapr/dapr/issues/3934)
+- **ADDED** Logging for mTLS cert expiration [3940](https://github.com/dapr/dapr/issues/3940)
+- **ADDED** Logging for sidecar pod name in Dapr Operator [3947](https://github.com/dapr/dapr/issues/3947)
+- **FIXED** Method invocation / Dapr automatically decodes %2f in URL [4008](https://github.com/dapr/dapr/issues/4008)
+- **FIXED** TLS connectivity to gRPC proxy functionality when --app-ssl is enabled [4048](https://github.com/dapr/dapr/issues/4048)
+- **ADDED** More metrics [4068](https://github.com/dapr/dapr/issues/4068)
+- **ADDED** logging for Dapr API calls over gRPC and HTTP [4119](https://github.com/dapr/dapr/issues/4119)
+- **ADDED** Nightly builds for Dapr [4124](https://github.com/dapr/dapr/issues/4124)
+- **FIXED** helm charts: increase failure threshold for liveness and readiness probes [4132](https://github.com/dapr/dapr/pull/4132)
+- **FIXED** dapr reconnect logic with rabbitmq results in "channel not initialized" [4143](https://github.com/dapr/dapr/issues/4143)
+- **ADDED** sidecar-injector health check port setting in helm chart [4174](https://github.com/dapr/dapr/issues/4174)
+- **FIXED** Configuration API, listen to all keys [4178](https://github.com/dapr/dapr/issues/4178)
+- **ADDED** HealthzPort option to injector helm chart - defaults to 8080 [4179](https://github.com/dapr/dapr/pull/4179)
+- **FIXED** Dapr retries forever when app is erroring on subscriptions [4185](https://github.com/dapr/dapr/issues/4185)
+- **FIXED** gRPC proxy connection closes after mTLS TTL passed [4197](https://github.com/dapr/dapr/issues/4197)
+- **ADDED** Dapr images on GitHub Registry in addition to Docker Hub [4199](https://github.com/dapr/dapr/issues/4199)
```

Comment:
```suggestion
- **ADDED** Publish Dapr images to GitHub container registry in addition to Docker Hub [4199](https://github.com/dapr/dapr/issues/4199)
```

Reply:
ADDED and Publish is confusing in the same sentence.

Reply:
Changed the rest.